src/pages/contact.tsx

```diff
@@ -0,0 +1,215 @@
+import { NextPage } from 'next'
+import { site } from '~/config'
+import Head from 'next/head'
+import { Blog, BlogPostRaw } from '~/modules/blog'
+import {
+  ContentManager,
+  PageProps,
+  StaticContent,
+  StaticPageMeta,
+} from '~/modules/content-manager'
+import { Button, PageLayout } from '../components'
+import Link from 'next/link'
+import { useForm, SubmitHandler } from 'react-hook-form'
+import { FormValues, sumbitForm } from './api/feedback'
+import classNames from 'classnames'
+import { useState } from 'react'
+
+export enum FeedbackState {
+  NOT_SENT,
+  SENT,
+  ERROR,
+}
+interface ContactProps {
+  meta: StaticPageMeta
+  posts: BlogPostRaw[]
+  contact: StaticContent[]
+}
+const Contact: NextPage<PageProps<ContactProps>> = ({ menu, social, data }) => {
+  const { meta, contact } = data
+  const { register, handleSubmit } = useForm<FormValues>()
+  const [feedbackState, setFeedbackState] = useState(FeedbackState.NOT_SENT)
+
+  const getFeedbackStateMessage = (state: FeedbackState) => {
+    switch (state) {
+      case FeedbackState.NOT_SENT:
+        return null
+      case FeedbackState.ERROR:
+        return (
+          <div className="bg-red-alert border-red-border p-8 text-dark-super font-bold border-t-2">
+            Type your message and email
```

Comment:
We need to handle different errors differently. If backend is down, you will show this message even if everything is filled up.

Reply:
I change to "Something went wrong"